install/operator/pkg/syndesis/backup/backup.go

```diff
@@ -290,15 +548,15 @@ func (b *Backup) clientConfig() *rest.Config {
 	return c
 }
 
-func (b *Backup) client() (c client.Client, err error) {
+func (b *Backup) client() client.Client {
 	if b.Client == nil {
 		cl, err := client.New(b.clientConfig(), client.Options{})
 		if err != nil {
-			return nil, err
+			util.ExitOnError(err)
 		}
 		b.Client = &cl
 	}
-	return *b.Client, nil
+	return *b.Client
 }
```

Comment:
I don't think this is a good approach and this is why. If the operator is scheduled to run a backup(and we can afford that a backup fails, because it is a backup right) and this method fails for whatever reason, then this will kill the operator.

Rather wrap the error and return I would say.